README.md

```diff
@@ -23,7 +23,7 @@ After you're finished please remove all the comments and instructions!
 -->
 
```

Comment:
Please give credit to the author of the original design `Gregoire Vella` on Behance as required by `the Creative Commons license of the design` in README `Acknowledgements` section. Please add `rspec` command in readme for the user to run tests.